roles/openshift_node_bootstrap_configmap/defaults/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+openshift_node_bootstrap_confimap_groups:
+- type: master
+  labels:
+  - 'type=master'
```

Comment:
As a note here, I opened #6011 because we need to be moving in the right direction with labels.  I would expect us to introduce the new "kube role label" pattern while preserving the old labels for some period of time.  But I think we're going to need to take away the choice (in our install process) to have a different pattern.